ExampleMod/Tiles/ExampleHerb.cs

```diff
@@ -6,79 +8,83 @@
 using static Terraria.ModLoader.ModContent;
 
 namespace ExampleMod.Tiles
-{
+{
+	//An enum on the 3 stages of herb growth.
+	public enum PlantStage : byte
+	{
+		Planted,
+		Growing,
+		Grown
+	}
+
+	//A plant with 3 stages, planted, growing and grown.
+	//Sadly, modded plants are unable to be grown by the flower boots
 	public class ExampleHerb : ModTile
 	{
-		public override void SetDefaults() {
+		private const byte FrameWidth = 18; //a field for readibilty and to kick out those magic numbers

```

Comment:
Should be int, but alright

Reply:
What's your reason why?

Reply:
Because it's just a number, the height of a frame. It has nothing to do with bytes.

Reply:
as far as I know it being a byte would reduce size

Reply:
You're not dealing with any networking or saving data here, so that doesn't matter.